### DISCUSSION TITLE
Ploting Warning

### DISCUSSION THREAD

**Discussion by hahahah-single** [GreenleafLab/ArchR#1056] [Questions / Documentation]
I am re-running my samples and get some warning when using plotFootprints and plotBrowserTrack function : guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.
I wondering how I can manage these warnings and can i ignore it ?

**Comment by rcorces** [comment]
This is a change implemented in newer versions of ggplot2. You can ignore the warning
